static/monorail-2.0.yaml

```diff
@@ -4767,8 +4767,6 @@ paths:
               $ref: '#/definitions/Error'
         tags:
         - /api/2.0
-        x-authentication-type:
```

Comment:
@tannoa2 You should probably remove the privileges too since they aren't used without authentication